### DISCUSSION TITLE
Screenshot

### DISCUSSION THREAD

**Discussion by kennynakamura** [sepinf-inc/IPED#738] [Ideas]
How do you do?
I was thinking and I noticed that on android screenshots are saved with the name "Screnshot...". Would it be useful to have a regex for the name of image files to filter screenshots?

**Comment by lfcnassif** [comment]
Hi @kennynakamura. I think this could result in many false positives, given that name is common out there.

**Comment by kennynakamura** [comment]
Hi! I found this project that can do something similar with screenshot chat, i guess this could be usefull.
I wrote a script and tested in iped-4.0.0, it's work fine.

https://github.com/kennynakamura/chatdectionIPED/blob/main/ChatDetectionTask.py

Project program and model link
https://github.com/Suji04/Chat_ScreenShot_Classifier
